Easydict/Feature/ViewController/Window/WindowManager/EZWindowManager.m

```diff
@@ -413,6 +413,14 @@ - (void)showFloatingWindow:(EZBaseQueryWindow *)window atPoint:(CGPoint)point {
         [preferencesWindowController.window close];
     }
     
+    // Workaround for SwiftUI Settings window
+    // https://github.com/tisfeng/Easydict/issues/362
+    for (NSWindow *window in [NSApp windows]) {
+        if ([window.identifier isEqualToString:@"com_apple_SwiftUI_Settings_window"]) {
```

Comment:
This code works, but can we find a better solution instead of hard coding?

Reply:
I'm not sure. IMHO, there is no better ways to handle the SwiftUI window in AppKit code.

Reply:
ok